challenge_4/python/OkraArmband/src/challenge_4.py

```diff
@@ -0,0 +1,49 @@
+class BinaryTree():
+    def __init__(self, root):
+        self.root = root
+        self.left = None
+        self.right = None
+
+    def invertTree(self):
+        """
+        Swap left and right children of input BinaryTree for all subsequent children as well (recursively).
+        """
+        temp = self.left
+        self.left = self.right
+        self.right = temp
+
+
+        # Continue the recursion as long as we haven't reached the end of a branch
+        if self.left is not None:
+            self.left.invertTree()
+        if self.right is not None:
+            self.right.invertTree()
+
+    @classmethod
+    def from_list(cls,data):
```

Comment:
If they pass in anything other than a number, yes. After all, it's possible for them to pass in a completely different object (say a dictionary, or  just another BST). If you did want to do the extra work, it'd be easier to check if what they've pass in is a number instead of checking for all the things that they shouldn't have passed in. For the purposes of this challenge, I don't think it's necessary.